deployment/updates/scripts/ini_files/audio_description_flavor_params.flavorParams.ini

```diff
@@ -26,4 +26,214 @@ deinterlice=0
 aspectRatioProcessingMode=0
 isGopInSec=0
 engineVersion=0
-type=1
\ No newline at end of file
+type=1
+
+[Audio-Description-French]
```

Comment:
As a general comment, why are these only in the update script and not in the base files used for a new install?
If you miss them there, it won't be added to a fresh envb install.

Reply:
Should the audio description flavors be on this separate file? the English AD is not on 04.flavorParams.ini